src/operator/controllers/otterizeclient/otterizegraphql/genqlient.graphql

```diff
@@ -24,4 +24,8 @@ query GetTLSKeyPair ($id: ID!) {
 
 mutation ReportComponentStatus($component: ComponentType!) {
     reportIntegrationComponentStatus(component: $component)
+}
+
+mutation CleanupOrphanK8SPodEntries($existingPodOwners: [NamespacedPodOwner!]!) {
```

Comment:
this should be called `reportActiveCertificateRequesters` now that you renamed the gql mutation. This applies also to the code in client.go. Pretty much everything should be renamed to "report" instead of "delete".